helpdesk/templates/helpdesk/ticket.html

```diff
@@ -197,7 +199,9 @@ <h5>{{ checklist }}</h5>
             </div>
         {% endif %}
 
+        {% if helpdesk_settings.HELPDESK_ENABLE_ATTACHMENTS %}
         <p id='ShowFileUploadPara'><button type="button" class="btn btn-warning btn-sm" id='ShowFileUpload'>{% trans "Attach File(s) &raquo;" %}</button></p>
```

Comment:
There are also related parts below in JS (line 278, and 307 to the end) that can be conditionally displayed.

Reply:
I hoped to get away without touching JS. But here we are.

Can you confirm that the following lines are the ones related to file handling?
- 278-282: `$("#ShowFileUpload")`...
- 307-334: `$('.add_file_fields_wrap)?`... and `$(document).on('change', ':file'`...

Reply:
Yes I confirm. These lines shouldn't trigger errors in console but it's better not to have them in DOM if attachments are disabled.